### DISCUSSION TITLE
Use CRUD without admin panel

### DISCUSSION THREAD

**Discussion by k-msalehi** [orchidsoftware/platform#2784] [Q&A]
I installed `orchid/crud` and `orchid/platform` and I only need to use it for CRUD operations without admin features like login, or admin template (admin sidebar). How is it possible?

**Comment by macit-emre** [comment]
orchid is an admin project. Here it says : https://orchid.software/en/docs/#introduction-to-laravel-orchid
There are many packages for crud operations that work standalone. Orchid uses its own way to develop inside admin panel.

**Comment by its4nas** [comment]
(Laravel CRUD Generator) It is a package that provides admin panel features; however, you can still use it solely for generating the CRUD operations without utilizing its admin features.

**Comment by k-msalehi** [comment]
> you can still use it solely for generating the CRUD operations without utilizing its admin features

@its4nas  Did you do it? I've tried but I couldn't use it just for CRUD, there is always an admin sidebar there, as I just want to use it CRUD feature without any admin template or other admin related feature,
